spec/integration/preview_spec.rb

```diff
@@ -32,7 +50,17 @@
         let (:template) { id }
         let (:fixture) { id }
 
-        specify { expect(response).to eq("content") }
+        specify { expect(response).to eq("{\"content\":\"as json\"}") }
+
+        expected_headers = {
+          "Content-Type"                => "application/json",
+          "Access-Control-Allow-Origin" => "*",
+          "X-Sequence"                  => expected_time,
+          "Content-Length"              => "21",
+          "X-Content-Type-Options"      => "nosniff"
+        }
+
+        specify { expect(last_response.headers).to eq(expected_headers) }
```

Comment:
I tend to agree 👍 

Charlie introduced the specify because it made the _output_ clearer, but I find myself very rarely looking at the output unless things start to error; and by that point I want to look through the specs to find a descriptive test identifier so I know what the test is supposed to be doing